bears/vcs/git/GitCommitBear.py

```diff
@@ -31,6 +31,18 @@ class GitCommitBear(GlobalBear):
     }
     CONCATENATION_KEYWORDS = [r',', r'\sand\s']
 
+    _nltk_data_downloaded = False
```

Comment:
No, not "only".

The variable is used to prevent the `download()` occurring twice.  `download()` isnt cheap; it fetches the index from the nltk website to determine whether the local cache needs updating.

While it is not likely that this bear will be invoked twice in typical coala usage, gitmate does run the bear on each commit in a PR.  `gitmate` probably invokes the coala command for each commit, rather than re-using the same runtime, but that is an example of how multiple instances of this global bear could be used in the same runtime.

This is essentially a very simple form of memoizing; so simple that even [functools.lru_cache](https://docs.python.org/3/library/functools.html#functools.lru_cache) is overkill (and doing a custom implementation is simpler to understand and simpler to test).

But I can still mock the tests, to avoid web fetches altogether, if that is requested.